deploy/operator.yaml

```diff
@@ -21,10 +21,13 @@ spec:
       containers:
         - name: cassandra-k8s-operator
           # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: jsanda/cassandra-k8s-operator:0.4.1-operator-metrics
```

Comment:
We'd rather keep the REPLACE_IMAGE line for the merge. This looks like your own testing image, isn't it?